test/integration-tests/strategies/delta-strategy-covered-call.ts

```diff
@@ -483,4 +482,55 @@ describe('Covered Call Delta Strategy integration test', async () => {
       expect(postReduceBal).to.be.lt(preReduceBal);
     });
   });
+  describe('end round2: assuming position got liquidated', async () => {
+    let strikes: BigNumber[];
+    let positionId: BigNumber;
+    before('set parameters', async () => {
+      strikes = await lyraTestSystem.optionMarket.getBoardStrikes(boardId);
+
+      positionId = await strategy.strikeToPositionId(strikes[3]);
+    });
+    it('liquidating position ', async () => {
+      await TestSystem.marketActions.mockPrice(lyraTestSystem, toBN('4300'), 'sETH');
+      const [positionBefore] = await lyraTestSystem.optionToken.getOptionPositions([positionId]);
+      expect(positionBefore.state).to.be.eq(PositionState.ACTIVE);
+      await lyraTestSystem.optionMarket.connect(randomUser).liquidatePosition(positionId, randomUser.address);
+
+      const [positionAfter] = await lyraTestSystem.optionToken.getOptionPositions([positionId]);
+      expect(positionAfter.state).to.be.eq(PositionState.LIQUIDATED);
+    });
+    it('fastforward to the expiry and settle options', async () => {
```

Comment:
Liquidated positions don't need to be settled as strategy will received remaining collateral back post liquidation. Since positions will be marked as status = LIQUIDATED, the vault will be able to close round without calling settlePosition (we just fixed the bug, so calling settlePosition on a liquidated position will not even be possible).